frontend/src/tests/session-tests.js

```diff
@@ -92,7 +94,7 @@ export function sessionsTests(api) {
         expect(updatedSession).toContainObject(newData);
```

Comment:
Don't know for sure, but should we call `apiGET` again and see if it match `updatedSession`? Maybe redundant, but just to be extra safe?

Reply:
I think I already did here, or am I wrong?

Reply:
Oops, did not see the folded part